crates/sui-types/src/object.rs

```diff
@@ -216,10 +216,24 @@ impl MoveObject {
         self.contents.splice(ID_END_INDEX.., value.to_le_bytes());
     }
 
+    /// Update the `timestamp_ms: u64` field of the `Clock` type.
+    pub fn set_clock_timestamp_ms(&mut self, timestamp_ms: u64) {
```

Comment:
should probably add more comments specifying how this needs to be a clock, plus make fn name `_unsafe`